erroranalysis/erroranalysis/_internal/matrix_filter.py

```diff
@@ -124,6 +124,9 @@ def compute_matrix_on_dataset(analyzer, features, dataset,
     for feature in features:
         if feature is None:
             continue
+        if feature not in analyzer.feature_names:
+            msg = 'Feature {} not found in dataset. Existing features: {}'
+            raise ValueError(msg.format(feature, analyzer.feature_names))
```

Comment:
Can we raise `UserConfigValidationException` instead of ValueError here. This should be available in the new raiutils package.

Reply:
done!